monitoring-custom-metrics/README.md

```diff
@@ -21,13 +21,18 @@ This example shows how to expose custom metrics to Prometheus with a Golang serv
     export KYMA_EXAMPLE_NS="{namespace}"
     ```
 
-2. Deploy the service:
+2. Ensure that your Namespace has Istio sidecar injection enabled. This example assumes that the metrics are exposed in a strict mTLS mode:
+
+   ````
+   kubectl label namespace ${KYMA_EXAMPLE_NS} istio-injection=enabled
+
+3. Deploy the service:
 
     ```bash
     kubectl apply -f deployment/deployment.yaml -n $KYMA_EXAMPLE_NS
     ```
 
-3. Deploy the ServiceMonitor:
+4. Deploy the ServiceMonitor:
 
     ```bash
     kubectl apply -f deployment/service-monitor.yaml

```

Comment:
In line 53 (can't add a comment there), replace `scrapped` with `scraped`.

[to scrap](https://www.merriam-webster.com/dictionary/scrap), scrapped: abandon, discard
[to scrape](https://www.merriam-webster.com/dictionary/scrape), scraped: to collect

Reply:
done